src/main/java/org/team199/robot2020/commands/DeployClimber.java

```diff
@@ -0,0 +1,52 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.team199.robot2020.commands;
+
+import org.team199.robot2020.Constants;
+import org.team199.robot2020.subsystems.Climber;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class DeployClimber extends CommandBase {
+  private Climber climber;
+
+  /**
+   * Deploys climber up to switch
+   */
+  public DeployClimber(Climber climber) {
+    addRequirements(this.climber = climber);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    climber.runLift(Constants.Climber.LIFT_DEPLOY_SPEED);
+    climber.runWinch(Constants.Climber.WINCH_DEPLOY_SPEED);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+      if (climber.getWinchHeight() >= 0) {
```

Comment:
When will winch height be 0?

Reply:
negative: winding out, positive: winding in. in Climber.java, I set the climber encoder's position to -90 when it's initialized